src/pages/nft/class/_classId.vue

```diff
@@ -294,6 +300,17 @@ export default {
         this.alertPromptError('LIKE_PRICE_IS_TEMPORARY_UNAVAILABLE');
       }
     },
+    handleCopyURL() {
+      const host = `${window.location.protocol}//${window.location.host}`;
+      const { path } = this.$route;
+      const URL = `${host}${path}?referrer=${this.getAddress}`;
```

Comment:
URL is a class 😂, might get confused
```suggestion
      const url = `${host}${path}?referrer=${this.getAddress}`;
```